app/src/main/java/com/matmoongi/FavoritesViewModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.matmoongi
+
+import androidx.lifecycle.ViewModel
+
+class FavoritesViewModel : ViewModel() {
+
+    private fun deleteInFavorites() {}
```

Comment:
뷰모델의 클래스 명에서 favorites이라는 건 알 수 있겠고, 추후에 파라메터가 추가될 테니,
```suggestion
    private fun unregister() {}
```
정도로만 해도 충분히 내용이 전달 될 것 같습니다.

Reply:
네 메서드가 속한 클래스를 통해서도 메서드의 동작을 유추할 수도 있겠네요